modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -1132,6 +1132,9 @@ export const console: ConsoleNS = {
                                 label: "Allowed grant type",
                                 validations: {
                                     empty: "Select at least a  grant type"
+                                },
+                                validation: {
+                                    refreshToken:"Refresh token grant type sholud be select with Code grant type."
```

Comment:
Could you check the other languages as well. we need this change in them as well.

Reply:
Address